.travis.yml

```diff
@@ -20,8 +20,6 @@ before_script:
   # prepare e2e test environment
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start" # the starting the virtual X frame buffer: Xvfb process
-  - sleep 3 #wait a while before xvfb to start
```

Comment:
@bhaskarmelkani we don't need to wait here because before running e2e tests we are linting and compiling, etc. "hopefully" selenium server has time to start.

Reply:
Yeah, we can try it.

Reply:
Yeah, we can try it.